src/FakeItEasy/A.cs

```diff
@@ -113,6 +113,7 @@ public static T Dummy<T>()
         /// <param name="typeOfDummy">The type of dummy to return.</param>
         /// <returns>A dummy object of the specified type.</returns>
         /// <exception cref="ArgumentException">Dummies of the specified type can not be created.</exception>
+        [EditorBrowsable(EditorBrowsableState.Advanced)]

```

Comment:
This should really have been added in the commit which added this new overload, but I'm happy to proceed as-is.

(Same applies to `CollectionOfDummy(Type typeOfDummy, int numberOfDummies)`.)